frontend/src/js/controllers/challengeListCtrl.js

```diff
@@ -50,7 +50,13 @@
                     }
 
                     if (vm.currentList[i].background_image === undefined || vm.currentList[i].background_image === null) {
-                        vm.currentList[i].background_image = vm.imgUrlObj.hulk;
+						if(typeof vm.imgUrlObj !== 'undefined' && typeof vm.imgUrlObj.hulk !== 'undefined') {
```

Comment:
@DakshMiglani In order to fix this you need to remove the code from line `52-54`, `88-90`, `124-127`. Actually, these are the old images that we were using earlier but currently we are not using them. :-)